__tests__/endpoints.test.js

```diff
@@ -51,3 +51,55 @@ describe("/api", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200 - Responds with a single article by article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
+      });
+  });
+  test("GET 404 - Article not found", () => {
+    return request(app).get("/api/articles/999").expect(404);
```

Comment:
I can see you do have both the finished error tests for this one but theyre not on this pr. Try and make sure youre doing it all together, rather than mixing and matching endpoints. It makes feedback harder